### DISCUSSION TITLE
Create a new wallet address using the CLI

### DISCUSSION THREAD

**Discussion by JWebCoder** [Chia-Network/chia-blockchain#5744] [Support]
Is is possible to create a new wallet address but keeping the same wallet using the CLI.
I know it's possible using the GUI, but i can't find a way to do it using the CLI.
Thank you.

**Comment by BrandtH22** [comment]
@JWebCoder , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - this is the cli command to derive more keys:
https://docs.chia.net/cli/#derive

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
